src/nomad_simulations/properties/energies.py

```diff
@@ -77,3 +78,502 @@ def __init__(
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class VDWEnergy(ClassicalEnergy):
+    """
+    Physical property section describing the van der Waals contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class ElectrostaticEnergy(ClassicalEnergy):
+    """
+    Physical property section describing all electrostatic contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class ElectrostaticShortRangeEnergy(ClassicalEnergy):
+    """
+    Physical property section describing short-range electrostatic contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class ElectrostaticLongRangeEnergy(ClassicalEnergy):
+    """
+    Physical property section describing long-range electrostatic contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class IntramolecularEnergy(ClassicalEnergy):
+    """
+    Physical property section describing all intramolecular contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class BondEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from bond interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class AngleEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from angle interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class DihedralEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from dihedral interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ImproperDihedralEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from improper dihedral interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ExternalEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from external interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
```

Comment:
I would consider this more of an energy flux, so the dimensionality is slightly different.

Reply:
This term is similar to the contributions coming from specific terms in the force field as defined above. Sometimes you define an external force within the force field (i.e., not linked to a particular atom type + config). Then you may want to again know the contribution to the total energy (e.g., per particle) from this external potential

Reply:
Well, it all depends on where you draw the boundary between your system and the outer world. If the latter interacts with the system, we can describe that as an energy flux, e.g.:

- a thermal bath: heat (a flux of kinetic and radiation energy) is being exchanged.
- an external electric or magnetic field: fluctuations enter the system as EM-waves, i.e. a flux of radiation energy.

Given these examples, we can represent the outer-world in simpler models, abstracting away a lot of complexity. However, I don't think most people consider them part of the system, so, in line with conservation of energy, they interact via a flux.

Reply:
With "external force," do you mean something like, e.g., a biasing potential or a wall in your box? Maybe this makes what we're talking about a bit clearer to @ndaelman-hu.

Reply:
Ow, that makes more sense! Yes, these can't be modeled via flux, as they are more so numerical / statistical tricks instead of physical forces (well, the biasing potential, at least).

I would definitely seek to associate these approaches with the method itself.